### ISSUE TITLE
Publish to NPM

### ISSUE THREAD

**Issue by ciclentfort** [alafr/SVG-to-PDFKit#158]
Hi, there have been a few fixes merged into the master branch since the last publish to npm. Any chance you could publish a new version to NPM?

**Comment by bluewire-solutions** [comment]
agree, it's also very misleading because SemVer is 0.1.8 for master branch and published npm version with some big changes

**Comment by AJLoveChina** [comment]
Hi @alafr , can you help on this?   Thanks

**Comment by jguse** [comment]
+1 @alafr 

My problem (which is fixed since one year in master): Want to support CMYK in `pdfmake` like

`<rect  [...] fill="cmyk(7, 0, 27, 0)"/>`

But `pdfmake` uses `svg-to-pdfkit@0.1.8`, which is the newest version...
I also tried with newest version and it works.
